lootsheetnpc5e.js

```diff
@@ -562,10 +562,16 @@ class LootSheet5eNPC extends ActorSheet5eNPC {
 
         priceModifier = Math.round(priceModifier * 100);
 
+        var maxPriceModifier = await game.settings.get("lootsheetnpc5e", "maxPriceModifier");
+
+        if (!maxPriceModifier){
```

Comment:
I wasn't 100% sure if you can do `var maxPriceModifier = await game.settings.get("lootsheetnpc5e", "maxPriceModifier") || 200;` with an async call, so stuck with a safer option